Harvest.Core/Services/SlothService.cs

```diff
@@ -165,10 +173,92 @@ public async Task<Result<SlothResponseModel>> MoveMoney(int invoiceId)
             return Result.Error("Sloth Response didn't have a success code for moneyTransfer id {transferId}: {data}", invoice.Id, data);
         }
 
+        private Result<bool> ProcessRefunds(TransactionViewModel model, decimal grandTotal, Invoice invoice)
+        {
+            var refundAmount = invoice.Expenses.Where(a => a.Total < 0).Sum(a => a.Total);
+            Log.Information("Refund amount should be negative here {refundAmount}", refundAmount);
```

Comment:
I think it's worth actually checking w/ an if statement.  If there is nothing to refund then we should just exit early.

Reply:
I'll fix in a sec.